tests/integ/modin/sql_counter.py

```diff
@@ -119,8 +119,10 @@ def __init__(
         high_count_reason=None,
         **kwargs,
     ) -> "SqlCounter":
+        from tests.integ.modin.conftest import SKIP_SQL_COUNT_CHECK
+
         self._queries: list[QueryRecord] = []
-        self._no_check = no_check
+        self._no_check = no_check or SKIP_SQL_COUNT_CHECK
```

Comment:
@sfc-gh-jdu if you also set self._no_check for the condition IS_IN_STORED_PROC also, I believe it should skip the Sql Counter and thus pytest assume

Reply:
yeah this is a good idea

Reply:
done